packages/dht/src/dht/DhtNode.ts

```diff
@@ -181,6 +182,12 @@ export class DhtNode extends EventEmitter<Events> implements ITransport {
         logger.trace(`Starting new Streamr Network DHT Node with serviceId ${this.config.serviceId}`)
         this.started = true
 
+        if (!isNodeJS()) {
```

Comment:
Would it make sense if we'd set these values outside of `DhtNode`? Now the object receives a config value in constructor but doesn't always apply that exact kind of configuration.  I.e. we could have a method `createDhtConfigDefaults` which would check the environment and pass in valid defaults?

(In that case we can drop also the check in at line `212`?)

Reply:
Could refactor the DhtNode's default config logic but not in this PR.